bitnami/wildfly/templates/NOTES.txt

```diff
@@ -1,13 +1,29 @@
 
 ** Please be patient while the chart is being deployed **
 
-{{- if .Values.ingress.enabled }}
+{{- if and .Values.exposeManagementConsole (or (eq "NodePort" .Values.service.type) (eq "NodePort" .Values.service.type) .Values.mgmtIngress.enabled) }}
```

Comment:
Duplicated `or (eq "NodePort" .Values.service.type) (eq "NodePort" .Values.service.type)`, is it a typo?